src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stringForConvert) {
+  const result = stringForConvert.trim().split(';');
```

Comment:
The variable name 'result' is not descriptive enough. Consider using a more descriptive name like 'styleDeclarations' to improve code readability.